dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/EnrollmentAnalyticsControllerTest.java

```diff
@@ -70,6 +71,8 @@ void setUp() {
   }
 
   @Test
+  @Disabled(
+      "generated query will fail on H2 database and we are now propagating BadSqlGrammarException since DHIS2-15184")
   void testGetQueryJson() {
```

Comment:
Now that we return a conflict message, we could assert the error message, instead of disabling it.